mods/ra/rules/misc.yaml

```diff
@@ -44,6 +44,7 @@ CRATE:
 		MaxAmount: 5
 		MinAmount: 1
 		MaxDuplicateValue: 1500
+		ValidTargets: GroundActor, WaterActor
```

Comment:
It should also include water actors

Reply:
Good catch, added them. (With the `Submarine` type not the under water type.)